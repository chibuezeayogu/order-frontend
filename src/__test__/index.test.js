/* eslint-disable no-undef */

import toJson from 'enzyme-to-json'
import Index from '../index'

describe('Render Dom', () => {
  it('renders without crashing', () => {
    expect(
      toJson({ ...Index, _reactInternalInstance: 'censored' })
    ).toMatchSnapshot()
  })
})
