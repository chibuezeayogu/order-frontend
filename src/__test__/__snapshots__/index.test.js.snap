// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Dom renders without crashing 1`] = `undefined`;
